packages/ember-environment/lib/global.js

```diff
@@ -14,5 +14,4 @@ function checkElementIdShadowing(value) {
 export default checkGlobal(checkElementIdShadowing(typeof global === 'object' && global)) ||
                checkGlobal(typeof self === 'object' && self) ||
                checkGlobal(typeof window === 'object' && window) ||
-               mainContext || // set before strict mode in Ember loader/wrapper
```

Comment:
Can you bring this back, and add it back to the correct location in `loader.js`? The ESLint cleanup PR should not have removed it (and I missed that it did when I reviewed).